src/util.rs

```diff
@@ -61,6 +62,7 @@ impl Drop for Mmap {
 }
 
 /// An owned file descriptor.
+#[derive(Debug)]
```

Comment:
The user can print `fd.0`.

Reply:
This impl was here so I could `#[derive(Debug)]` on types that contain `Fd`s, avoiding manual implementations in many cases.

Reply:
I don't recommend users to use `types::Fd` directly, it is basically only used for opcode construction.

Reply:
This is `util::Fd` not `types::Fd`. `util::Fd` is contained by `IoUring` and allows it to derive `Debug`. `types::Fd` already implemented `Debug` before this change.

Reply:
Sorry, I read it wrong. :'(